Atlas/android/app/src/main/java/com/bounswe2017/group10/atlas/profile/OwnItemActivity.java

```diff
@@ -27,17 +28,39 @@ protected void onCreate(@Nullable Bundle savedInstanceState) {
         setContentView(R.layout.activity_ownitem);
         Toolbar toolbar = findViewById(R.id.home_toolbar);
         setSupportActionBar(toolbar);
-        setTitle("My Items");
         this.mActionBar = getSupportActionBar();
         mActionBar.setDisplayHomeAsUpEnabled(true);
 
+        Bundle b = getIntent().getExtras();
+        int value = -1; // or other values
+        if(b != null)
+            value = b.getInt("type");
+
+        if(value==1)
```

Comment:
These value integers could be final constants like MY_ITEMS and MY_FAVORITES in the Constants.java